src/Pages/SingleSummaryPage/SingleSummaryPage.js

```diff
@@ -1,5 +1,25 @@
+import CardWrapper from '../../Atoms/CardWrapper/CardWrapper';
+import LeaderboardList from './../../Molecules/LeaderboardList/LeaderboardList';
+import MainButton from './../../Atoms/MainButton/MainButton';
+import PenaltyOrDisqualifiedLine from '../../Atoms/Leaderboard/PenaltyOrDisqualifiedLine';
+import SummaryLine from '../../Atoms/Leaderboard/SummaryLine';
+import WYNIK_RANDOMOWEGO_PSA from './../../Data/Dummy-data/RandomDogResult';
+
+const sendDataHandler = () => {};
+
 const SingleSummaryPage = () => {
-  return <h1>I am a SingleSummary Page Component</h1>;
+  return (
+    <CardWrapper column>
+      <LeaderboardList result={WYNIK_RANDOMOWEGO_PSA} />
+      <PenaltyOrDisqualifiedLine result={WYNIK_RANDOMOWEGO_PSA} />
+      <SummaryLine result={WYNIK_RANDOMOWEGO_PSA} />
```

Comment:
maybe it will be faster and less logic lower, when in that page will be countig sum of points and then sending by props. And whole array just where it is needed?

Reply:
Could be, but the page itself is not the part of the scope (?)
It seems only one ModalPage has some more logic ony the page.
Maybe we should decide in a team?